CHANGELOG/CHANGELOG-1.6.md

```diff
@@ -0,0 +1,114 @@
+# Changelog 1.6
+
+## 1.6.0 - 2022-03-29
+
+The Egress feature is graduated from Alpha to Beta and is therefore enabled by default.
+
+### Added
+
+- Add support for the following capabilities for [Antrea IPAM] feature:
+  * Support pre-allocating continuous IPs for StatefulSet. ([#3281](https://github.com/antrea-io/antrea/pull/3281), [@annakhm])
+  * Support specifying VLAN for IPPool. Traffic from Pods whose IPPools are configured with a VLAN ID will be tagged when leaving the Node uplink. ([#3247](https://github.com/antrea-io/antrea/pull/3247), [@gran-vmv])
+- Add support for the following capabilities for [Antrea Multi-cluster] feature:
+  * Support Antrea ClusterNetworkPolicy replication in a ClusterSet. ([#3363](https://github.com/antrea-io/antrea/pull/3363), [@Dyanngg])
+  * Add `antctl mc` subcommand for Antrea Multi-cluster resources. ([#3341](https://github.com/antrea-io/antrea/pull/3341) [#3287](https://github.com/antrea-io/antrea/pull/3287), [@hjiajing] [@bangqipropel])
+- Add support for the following capabilities for [AntreaPolicy] feature:
+  * Add Node selector in Antrea-native policies to allow matching traffic originating from or destined for Nodes. ([#3038](https://github.com/antrea-io/antrea/pull/3038), [@wenqiq])
+  * Add ServiceAccount selector in Antrea-native policies to allow selecting Pods by ServiceAccount. ([#3044](https://github.com/antrea-io/antrea/pull/3044), [@GraysonWu])
+  * Support Pagination for ClusterGroupMembership. ([#3183](https://github.com/antrea-io/antrea/pull/3183), [@qiyueyao])
+  * Add Port Number to Audit Logging. ([#3277](https://github.com/antrea-io/antrea/pull/3277), [@qiyueyao])
+- [Flow Visibility] Add Grafana Flow Collector as the new visualization tool for flow records.
+  * Add Grafana dashboards, Clickhouse data schema, deployment files, and doc. ([#3063](https://github.com/antrea-io/antrea/pull/3063) [#3525](https://github.com/antrea-io/antrea/pull/3525), [@heanlan] [@zyiou] [@dreamtalen])
+  * Add support for exporting flow records to ClickHouse from Flow Aggregator. ([#3196](https://github.com/antrea-io/antrea/pull/3196) [#3526](https://github.com/antrea-io/antrea/pull/3526), [@wsquan171] [@dreamtalen])
+  * Add ClickHouse monitor to ensure data retention for in-memory ClickHouse deployment. ([#3244](https://github.com/antrea-io/antrea/pull/3244) [#3498](https://github.com/antrea-io/antrea/pull/3498), [@yanjunz97])
+- [Multicast] Support IGMPv3 leave action. ([#3389](https://github.com/antrea-io/antrea/pull/3389), [@wenyingd])
+- [Windows] Add support for EndpointSlices on Windows Nodes. ([#3321](https://github.com/antrea-io/antrea/pull/3321), [@XinShuYang])
+- Add SKIP_CNI_BINARIES environment variable to support skipping installing specified CNI plugins. ([#3454](https://github.com/antrea-io/antrea/pull/3454), [@jainpulkit22])
+- Support antrea-agent UBI8 based image. ([#3273](https://github.com/antrea-io/antrea/pull/3273), [@ksamoray])
```

Comment:
technically we have a single image, so it's not specific to the antrea-agent, it can be used to run the controller as well

so maybe: Support UBI8-based container image to run Antrea